File: API.PONCHADOR/Modelo/Usuarios.cs
using System;

namespace API.PONCHADOR.Modelo
{
    public class Usuarios
    {
        public string Id { get; set; }
        public string Usuario { get; set; }
        public string Nombre { get; set; }
        public string Apellido { get; set; }
        public string Clave { get; set; }
        public int perfil { get; set; }
        public DateTime Fecha { get; set; }
        public int Estado { get; set; }
        public string Cedula { get; set; }
        public int Modelo { get; set; }
        public int CargoEmpleadosId { get; set; }
        public int RolUsuarioId { get; set; }
    }
}

File: nomina_dk (1)/nomina_dk/src/Modelo/Usuarios.ts


export interface Usuarios{
    Usuario: string | null;
    Nombre: string | null;
    Apellido: string | null;
    Clave: string | null;
    perfil: number | 0;
    Fecha: Date | null;
    Estado: number | 0;
    Cedula: string | null;
    Modelo: number | 0;
    CargoEmpleadosId: number | 0;
    RolUsuarioId: number | 0;
}